opencti-platform/opencti-front/src/private/components/analyses/groupings/GroupingKnowledgeGraphBar.jsx

```diff
@@ -986,10 +986,10 @@ class GroupingKnowledgeGraphBar extends Component {
                         fromObjects={relationFromObjects}
                         toObjects={relationToObjects}
                         startTime={
-                          lastLinkFirstSeen || dateFormat(grouping.published)
+                          lastLinkFirstSeen || validStartTimeForRelationCreation(now())
                         }
                         stopTime={
-                          lastLinkLastSeen || dateFormat(grouping.published)
+                          lastLinkLastSeen || now()
```

Comment:
Calling 2 times function `now` results having two different dates. Yes the diff would be only some milliseconds. But I think it's better to have only one call to `now` to be sure we have the same date. (Same remark for other files)